src/AggregationFunction/FunctionSum.php

```diff
@@ -63,7 +62,7 @@ public function getFilterDataType()
 	 */
 	public function processDataSource($dataSource)
 	{
-		if ($dataSource instanceof DibiFluent) {
+		if ($dataSource instanceof Dibi\Fluent || $dataSource instanceof DibiFluent) {
```

Comment:
Will that work when there is no `use DibiFluent;` statement in the file header?

Reply:
@cniry ping

Reply:
I think it is at least missing "use Nette" OR the test should be "$dataSource instanceof \Dibi\Fluent ..." (without any of these changes it doesn't work for me).